package.json

```diff
@@ -11,7 +11,7 @@
     "firefoxos",
     "testing"
   ],
-  "version": "1.16.0-beta.3",
+  "version": "1.16.0-rc.1",
```

Comment:
is this needed?

Reply:
@mykola-mokhnach the next will be to hack off the `-rc.1` suffix and then tag + publish 1.16.0.